dev/openchpl_preload.sql

```diff
@@ -1302,14 +1302,12 @@ INSERT INTO openchpl.muu_accurate_as_of_date (accurate_as_of_date, last_modified
 INSERT INTO user_permission (user_permission_id, "name", description, authority, last_modified_user) VALUES
 (-2, 'ADMIN', 'This permission confers administrative privileges to its owner.', 'ROLE_ADMIN', -1),
 (1, 'USER_CREATOR' ,'This permission allows a user to create other users', 'ROLE_USER_CREATOR' , -1),
-(2, 'ACB_ADMIN' ,'This permission gives a user write access to their ACBs.', 'ROLE_ACB_ADMIN' , -1),
-(3, 'ACB_STAFF' ,'This permission gives a user read access to their ACBs', 'ROLE_ACB_STAFF' , -1),
-(4, 'ATL_ADMIN' ,'This permission gives a user write access to their ATLs.', 'ROLE_ATL_ADMIN' , -1),
-(5, 'ATL_STAFF' ,'This permission gives a user write access to their ATLs.', 'ROLE_ATL_STAFF' , -1),
+(2, 'ACB' ,'This permission gives a user write access to their ACBs.', 'ROLE_ACB_ADMIN' , -1),
+(4, 'ATL' ,'This permission gives a user write access to their ATLs.', 'ROLE_ATL_ADMIN' , -1),
 (6, 'CMS_STAFF' ,'This permission gives a user read access to CMS reports.', 'ROLE_CMS_STAFF' , -1),
 (7, 'ONC_STAFF' ,'This permission gives a user access to the CMS Download file and report navigation section. It denies editing of Users/Products/CPs/ACBs/etc. No user invitation ability.', 'ROLE_ONC_STAFF' , -1);
 
-SELECT pg_catalog.setval('user_permission_user_permission_id_seq', 8, true);
+SELECT pg_catalog.setval('user_permission_user_permission_id_seq', 6, true);
```

Comment:
Setting this to 6 means that the next roles that are added would overwrite the `CMS_STAFF` and `ONC_STAFF` roles